.git-hooks/hooks/pre-commit

```diff
@@ -1,36 +0,0 @@
-#!/bin/bash
```

Comment:
Why is this file deleted?

Reply:
Because my computer does not let me push changes if I have that file. It's specific to my device so I just get rid of it, hence I said only look at adcs files.

Reply:
I can't just look at adcs files. When this is merged it brings ALL changes, not just the ones you wanted me to see. Also, your computer does not let you push on purpose, you need to install clang-format